languages/sr_RS/Zend_Validate.php

```diff
@@ -0,0 +1,308 @@
+    "The input is an empty string" => "Унет је празан текст",
+
+    // Zend\I18n\Validator\Alpha
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+    "The input contains non alphabetic characters" => "Унос садржи не-алфабетске карактере",
+    "The input is an empty string" => "Унет је празан текст",
+
+    // Zend\I18n\Validator\DateTime
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+    "The input does not appear to be a valid datetime" => "Унет је неисправан датум и време",
+
+    // Zend\I18n\Validator\IsFloat
+    "Invalid type given. String, integer or float expected" => "Погрешан тип унет. Очекиван је текст, цео број или стварни број",
+    "The input does not appear to be a float" => "Унос није стварни број",
+
+    // Zend\I18n\Validator\IsInt
+    "Invalid type given. String or integer expected" => "Погрешан тип унет. Очекиван је текст или цео број",
+    "The input does not appear to be an integer" => "Унос није цео број",
+
+    // Zend\I18n\Validator\PhoneNumber
+    "The input does not match a phone number format" => "Унос не одговара формату телефонског броја",
+    "The country provided is currently unsupported" => "Одабрана земља тренутно није подржана",
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+
+    // Zend\I18n\Validator\PostCode
+    "Invalid type given. String or integer expected" => "Погрешан тип унет. Очекиван је текст или цео број",
+    "The input does not appear to be a postal code" => "Унос није поштански број",
+    "An exception has been raised while validating the input" => "Дошло је до грешке приликом провере уноса",
+
+    // Zend\Validator\Barcode
+    "The input failed checksum validation" => "Унос није прошао проверу вредности",
+    "The input contains invalid characters" => "Унос садржи неисправне карактере",
+    "The input should have a length of %length% characters" => "Унос треба да има дужину од %length% карактера",
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+
+    // Zend\Validator\Between
+    "The input is not between '%min%' and '%max%', inclusively" => "Унос није између '%min%' и '%max%', инклузивно",
+    "The input is not strictly between '%min%' and '%max%'" => "Унос није искључиво између '%min%' и '%max%'",
+
+    // Zend\Validator\Bitwise
+    "The input has no common bit set with '%control%'" => "Унос нема подразумевани бит подешен са '%control%'",
+    "The input doesn't have the same bits set as '%control%'" => "Унос нема исте бите подешене као '%control%'",
+    "The input has common bit set with '%control%'" => "Унос има подразумевани бит подешен са '%control%'",
+
+    // Zend\Validator\Callback
+    "The input is not valid" => "Унос је неисправан",
+    "An exception has been raised within the callback" => "Дошло је до грешке приликом позива",
```

Comment:
IMHO "callback" in this case should be translated more precisely, because this error is about PHP callback function, and not any callback. I would go with something like "Došlo je do greške u toku izvršavanja pozvane funkcije".